sensor/kubernetes/eventpipeline/message/message.go

```diff
@@ -0,0 +1,52 @@
+package message
+
+import (
+	"github.com/stackrox/rox/generated/internalapi/central"
+	"github.com/stackrox/rox/generated/storage"
+)
+
+// CompatibilityDetectionMessage should be used by old handlers
+// it's here for retrocompatibility reasons.
+type CompatibilityDetectionMessage struct {
+	// Object The deployment object that needs to be processed by the detector
+	Object *storage.Deployment
+	// Action the event action (CREATE_RESOURCE, REMOVE_RESOURCE, UPDATE_RESOURCE, and SYNC_RESOURCE)
+	Action central.ResourceAction
+}
+
+// ResourceEvent message used by the event pipeline's components
+type ResourceEvent struct {
+	// ForwardMessages messages generated by the handlers that need to be forwarded to central
+	ForwardMessages []*central.SensorEvent
+	// CompatibilityDetectionDeployment should be used by old handlers
+	// and it's here for retrocompatibility reasons.
+	// This property should be removed in the future and only the
+	// DetectionObject should be sent
+	CompatibilityDetectionDeployment []CompatibilityDetectionMessage
+	// ReprocessDeployments is also used for compatibility reasons with Network Policy handlers
+	// in the future this will not be needed as the dependencies are taken care by the resolvers
+	ReprocessDeployments []string
+}
+
+// WrapOutputMessage wraps the SensorEvents, CompatibilityDetectionMessages, and the ReprocessDeployments into a ResourceEvent message
+func WrapOutputMessage(sensorMessages []*central.SensorEvent, detectionDeployment []CompatibilityDetectionMessage, reprocessDeploymentsIds []string) *ResourceEvent {
```

Comment:
Thanks for moving this functions out. I think it's a lot better to have them here.